packages/upstream-extra/ocamlformat-rpc-lib.0.26.2/opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "ocamlformat-rpc-lib"
-version: "0.26.1"
+version: "0.26.2"
```

Comment:
Do we need to update .ocamlformat anywhere?

Reply:
No, because it introduced a lot of friction at the time. There are not formatting changes between the versions